src/app/chip_data_model.gni

```diff
@@ -390,6 +390,11 @@ template("chip_data_model") {
           "${_app_root}/clusters/${cluster}/thread-network-diagnostics-provider.cpp",
           "${_app_root}/clusters/${cluster}/thread-network-diagnostics-provider.h",
         ]
+      } else if (cluster == "thread-border-router-management-server") {
+        sources += [
+          "${_app_root}/clusters/${cluster}/${cluster}.cpp",
```

Comment:
Please list all the files of the cluster impl.  Two are missing.

Reply:
Added